README.md

```diff
@@ -107,25 +107,20 @@ When wishing to leverage feature flagging within the local pod, the following st
 _See [here](config/samples/crds/custom_provider.yaml) for additional custom resource parameters_
 
 ```
-apiVersion: core.openfeature.dev/v1alpha1
+apiVersion: core.openfeature.dev/v1beta1
```

Comment:
Can we not move to `v1alpha2` ? going to beta starts to lower the amount of wiggle room we have, the next API version will be graduated to `v1`